apps/search-poc/src/components/ResourceDetails/__tests__/__snapshots__/ResourceDeatils.test.tsx.snap

```diff
@@ -2,60 +2,40 @@
 
 exports[`ResourceDetails renders correctly 1`] = `
 <div
-  className="ResourceDetails"
+  className="resource-details undefined"
```

Comment:
I know you're not testing your code but when I look at that, there is obviously an issue. If not class names are passed downs as props we need to make sure we don't get `undefined` even though in reality this isn't going to break anything...

Reply:
Can you maybe have a look and fix the code too in this PR?

Reply:
Updated.